### DISCUSSION TITLE
Piper text cut off via home assistant

### DISCUSSION THREAD

**Discussion by PetePeter** [rhasspy/piper#633] [General]
Piper running on my home assistant (HasOS) tends to cut off phrases.
If I want to say "Hello person at the door"
i actually hear: "Hello person at"

this happens with the destination is a reolink doorbell camera, or a tapo c200 camera.
why?

I googled around and couldnt find a solution :(
Thank you

**Comment by UsefulVid** [comment]
I could solve this by follow the piper command with a "delay" command to stop HA doing anything else with the speaker.

**Reply by PetePeter** [reply]
ah i ended up solving it by using chime to do the delay. i failed to do the piper delay 

but chime is great as different cameras need different delays

ie tapo & reolink
